docs/USAGE.md

```diff
@@ -30,7 +32,9 @@ java -jar gtfs-validator-v2.0.jar --input relative/path/to/dataset.zip --output
 ...which will:
  1. Search for a GTFS dataset located at `relative/path/to/dataset.zip`
  1. Validate the GTFS data and output the results to the directory located at `relative/output/path`. 
- 1. Export both validation and system errors reports to JSON by default. This folder will contain the `.json` file with information related to the validation process. The validation report will be named as `report.json` and the system errors report can be found under the name of `system_errors.json`. 
+ 1. Export both validation and system errors reports to JSON by default. This folder will contain the `.json` file with information related to the validation process. The validation report will (by default) be named as `report.json` and the system errors report can be found under the name of `system_errors.json`.
+ 
+  ⚠️ Note that reports naming can be overridden by providing values to `-v` and/or `-e` CLI arguments. These **must** include `.json` file extension.
```

Comment:
Does it absolutely need to have a .json extension? I could see cases where someone might want to change the extension too. If it still functions without a .json extension I would change the "must" to a "should".

Reply:
It actually works with other extensions `.txt` for example. Modifying as follows (in order to make it clear that other extensions can be used too).

```suggestion
  ⚠️ Note that reports naming can be overridden by providing values to `-v` and/or `-e` CLI arguments. These **must** include the file extension.
```

Reply:
Can you use it without a file extension? I could see someone wanting to simulate an API endpoint like `https://.../agency1/results`, in which case you wouldn't provide an extension.

Reply:
It does. Thanks for the clarification, modifying as you suggest.